src/Heist/Splices/Json.hs

```diff
@@ -214,6 +234,22 @@ explodeTag = ask >>= go
         "value"    ## varAttrTag v valueTag
 
 
+#if MIN_VERSION_aeson(2,0,0)
+    --------------------------------------------------------------------------
+    goObject :: KM.KeyMap Value -> ReaderT Value (HeistT n n) Template
+    goObject obj = do
+        start <- genericBindings
+        let bindings = FI.ifoldl' bindKvp start  obj
```

Comment:
There is ways to make the code `aeson-2` compatible with less duplication of code.